Source/WebCore/html/MediaElementSession.cpp

```diff
@@ -1241,14 +1241,16 @@ void MediaElementSession::didReceiveRemoteControlCommand(RemoteControlCommandTyp
 
 bool MediaElementSession::hasNowPlayingInfo() const
 {
```

Comment:
I think this would greatly reduce the current user interface when something is playing. Right now it shows in the Lock Screen (on iOS) or in the control centre on Mac. Following this it won't show at all.

Isn't there a way to prevent automatic play/pause (such as when removing the AirPods) but would still let you interact with the video controls if you do want to pause or resume ? or even get some details about what is playing?

This seems like a really unfortunate side-effect that just to prevent removing the AirPods to pause the video, to prevent all other interactions while at it

Reply:
Without more OS support, it is difficult to make it work properly by default without web page help.
That said, I'll restrict these changes to macOS for now.